_posts/2020-09-14-release-0-21-4.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: post
+title:  "JuMP 0.21.4 is released"
+date:   2020-09-14
+categories: [releases]
+permalink: /blog/0.21.4-release/
+---
+
+We are happy to announce the joint release of JuMP 0.21.4 and MathOptInterface
+0.9.15! These releases are a mix of new features and some much needed
+performance optimizations. This should be a non-breaking release, please let us
+know if this isn't the case by [opening a Github issue](https://github.com/jump-dev/JuMP.jl/issues/new/choose)
+or joining the [Gitter chat](https://gitter.im/JuliaOpt/JuMP-dev).
+
+On the JuMP side, new features of interest include
+* the `relax_integrality` function, which relaxes all integrality constraints in
+the model. This is useful for solving the LP relaxation of a MIP.
+* the `compute_conflict!` function, which exposes the ability of some solvers to
+compute an irreducible infeasible subsystem of an infeasible model. This is very
+useful when debugging, and we expect more work in the future to refine a
+user-friendly API.
```

Comment:
Are we implying that the current API isn't user friendly? Maybe delete "This is very useful ..." and the text that follows (it doesn't add much essential information).